### DISCUSSION TITLE
does qb send x-client data?

### DISCUSSION THREAD

**Discussion by al-caveman** [qutebrowser/qutebrowser#5703] [Q&A] [answered]
https://9to5google.com/2020/02/06/google-chrome-x-client-data-tracking/

**Comment by Stas1981ru** [comment]
We can try to see that when connected to google site by http proxy.
In proxy log we can see all headers, which QWebEngineView transmit.

**Comment by The-Compiler** [comment, marked as answer]
I did some testing with Google Chrome 85.0.4183.83, Chromium 85.0.4183.83 and qutebrowser (QtWebEngine 5.15.0):

- Google Chrome does not send the header by default, only if experimental features are turned on in `chrome://flags`
- Chromium does never send the header (even if experimental features are turned on)
- qutebrowser does never send the header (it doesn't have `chrome://flags` anyways)
